### ISSUE TITLE
Does not work with macOS

### ISSUE THREAD

**Issue by potmo** [Amzd/ScrollViewProxy#4]
The `Package.swift` states that the package should work with macOS (`.macOS(.v10_13),`) but since `UIScrollView` does not exist on macOS (it's called `NSScrollView` there instead) it doesn't.

Introspect seems to provide `NSScrollView` on macOS so maybe its just a quick fix but I don't know how much `NS` and `UI` APIs differ. With a bit of luck its only [this declaration](https://github.com/Amzd/ScrollViewProxy/blob/master/Sources/ScrollViewProxy/ScrollViewProxy.swift#L51) that breaks it

**Comment by potmo** [comment]
I fiddled around some today and I'm having problems with introspects min requirement for macOS `15.0`. I think it should be `10.15` (as in Catalina).  I dropped a bug [here](https://github.com/siteline/SwiftUI-Introspect/issues/46).
Apart from that it seems to be a few things named differently but I think it might work. I'm probably giving up on it since I dont need iOS and I guess its easier to just make a macOS version for this
